docs/getting-started/local-development.mdx

```diff
@@ -1,65 +1,98 @@
+to protect against CSRF attacks. We recommend taking a deep-dive into [Ory's security model](../security-model.mdx) if you are not
+familiar with the concepts.
+
+Ory also supports advanced SSO capabilities based on OpenID Connect. For system that do not require an enterprise SSO solution, we
+recommend sticking to session cookies.
+
+When developing locally, your application is often exposed at the `localhost` domain. Ory APIs must be available on the same
+domain (so `localhost`) as your application to avoid issues with third-party cookie (e.g. when using Ory Session Cookies). The
+easiest way to expose Ory on your `localhost` domain is to use the [Ory CLI Tunnel](../guides/cli/20_proxy-and-tunnel.mdx).
+
+Copy the project slug from your project's [Connect page](https://console.ory.sh/projects/current/connect) and start the Ory
+Tunnel. Here we assume that your app runs on `http://localhost:3000`:
+
+```shell
+ory tunnel --dev --project {project.id-or-project.slug} \
+  http://localhost:3000
+```
+
+Running this commands opens the `4000` port on your local machine. The URL `http://localhost:3000` is where your application is
+available. The `--dev` flag disables a few security checks to make local development easier. After starting the Tunnel, test it by
+performing the registration flow: [`http://localhost:4000/ui/registration`](http://localhost:4000/ui/registration).
+
+You can now start developing locally. Make sure to send any HTTP calls you make to `http://localhost:4000` and not
+`https://<your-slug>.projects.oryapis.com`. For example, when you configure the Ory SDK (here in Node.js):
+
+```typescript
+import { Configuration, FrontendApi } from "@ory/client"
+
+const frontend = new FrontendApi(
   new Configuration({
-    basePath: "http://localhost:4000", // instead of `https://<your-slug>.projects.oryapis.com`
+    // Replace
+    //  basePath: "https://{project.slug}.projects.oryapis.com",
+    // with
+    basePath: "http://localhost:4000",
     baseOptions: {
       withCredentials: true,
     },
   }),
 )
-```
-
-## Limitations
-
-Using the Ory Tunnel has a few limitations.
 
-### `localhost` vs `127.0.0.1`
-
-Your local machine is usually available on both `localhost` and `127.0.0.1`. For the browser these two domains are different
-top-level-domains. Cookies set on `localhost` will not be sent in requests to `127.0.0.1` and vice versa.
+const { data: login } = await frontend.createBrowserLoginFlow()
+const { data: session } = await frontend.toSession()
+// ...
+```
 
-:::tip
+Only the `FrontendApi` needs to access the Ory Tunnel. Administrative APIs like `IdentityApi` can be accessed directly from your
+project slug, although they also work via the Ory Tunnel.
 
-If you encounter cookie issues, make sure that all of your requests are sent to `localhost` and not `127.0.0.1`.
+### Limitations
 
-:::
+Using the Ory Tunnel has two key limitations currently.
```

Comment:
Simplify. Remove words "first" and "second". Bring back list formatting, use either UL or OR.